Makefile

```diff
@@ -70,25 +70,54 @@ create-awsfederatedaccountaccess: check-aws-account-id-env
 	# Create awsFederatedAccountAccess CR
 	@oc process -p AWS_IAM_ARN=${AWS_IAM_ARN} -p IAM_USER_SECRET=${IAM_USER_SECRET} -p AWS_FEDERATED_ROLE_NAME=${AWS_FEDERATED_ROLE_NAME} -p NAMESPACE=${NAMESPACE} -f hack/templates/aws_v1alpha1_awsfederatedaccountaccess_cr.tmpl | oc apply -f -
 	# Wait for awsFederatedAccountAccess CR to become ready
-	@while true; do STATUS=$$(oc get awsfederatedaccountaccess -n aws-account-operator test-federated-user -o json | jq -r '.status.state'); if [ "$$STATUS" == "Ready" ]; then break; elif [ "$$STATUS" == "Failed" ]; then echo "awsFederatedAccountAccess CR test-federated-user failed to create"; exit 1; fi; sleep 1; done
+	@while true; do STATUS=$$(oc get awsfederatedaccountaccess -n ${NAMESPACE} test-federated-user -o json | jq -r '.status.state'); if [ "$$STATUS" == "Ready" ]; then break; elif [ "$$STATUS" == "Failed" ]; then echo "awsFederatedAccountAccess CR test-federated-user failed to create"; exit 1; fi; sleep 1; done
 	# Print out AWS Console URL
-	@echo $$(oc get awsfederatedaccountaccess -n aws-account-operator test-federated-user -o json | jq -r '.status.consoleURL')
+	@echo $$(oc get awsfederatedaccountaccess -n ${NAMESPACE} test-federated-user -o json | jq -r '.status.consoleURL')
 	# Wait ${SLEEP_INTERVAL} seconds for AWS to register role
 	@sleep ${SLEEP_INTERVAL}
 
 .PHONY: test-switch-role
 test-switch-role:
 	# Retrieve role UID
-	$(eval UID=$(shell oc get awsfederatedaccountaccesses.aws.managed.openshift.io -n aws-account-operator test-federated-user -o=json |jq -r .metadata.labels.uid))
+	$(eval UID=$(shell oc get awsfederatedaccountaccesses.aws.managed.openshift.io -n ${NAMESPACE} test-federated-user -o=json |jq -r .metadata.labels.uid))
 	# Test Assume role
 	aws sts assume-role --role-arn arn:aws:iam::${OSD_STAGING_1_AWS_ACCOUNT_ID}:role/read-only-$(UID) --role-session-name RedHatTest --profile osd-staging-2
 
 # Delete awsFederatedAccountAccess
 # This uses a AWS Account ID from your environment
 .PHONY: delete-awsfederatedaccountaccess
 delete-awsfederatedaccountaccess: check-aws-account-id-env
-	# Delete federatedaccountaccess with secret
+# Delete federatedaccountaccess with secret
 	@oc process -p AWS_IAM_ARN=${AWS_IAM_ARN} -p IAM_USER_SECRET=${IAM_USER_SECRET} -p AWS_FEDERATED_ROLE_NAME=${AWS_FEDERATED_ROLE_NAME} -p NAMESPACE=${NAMESPACE} -f hack/templates/aws_v1alpha1_awsfederatedaccountaccess_cr.tmpl | oc delete -f -
 
 .PHONY: test-awsfederatedaccountaccess
 test-awsfederatedaccountaccess: check-aws-account-id-env create-account create-awsfederatedrole create-awsfederatedaccountaccess test-switch-role delete-awsfederatedaccountaccess delete-awsfederatedrole delete-account
+
+.PHONY: deploy-aws-account-operator-credentials
+deploy-aws-account-operator-credentials:
+# Base64 Encode the AWS Credentials
+	$(eval ID=$(shell base64 <<< ${OPERATOR_ACCESS_KEY_ID}))
+	$(eval KEY=$(shell base64 <<< ${OPERATOR_SECRET_ACCESS_KEY}))
```

Comment:
These commands need to be changed to the `echo -n ${OPERATOR_ACCESS_KEY_ID} | base64` format otherwise it adds the newline problem I ran into earlier.

Reply:
Niiiiiice catch @iamkirkbater!

Fixed!